charts/prometheus-elasticsearch-exporter/values.yaml

```diff
@@ -18,13 +18,25 @@ image:
   pullPolicy: IfNotPresent
   pullSecret: ""
 
-## Set enabled to false if you don't want securityContext
-## in your Deployment.
-## The below values are the default for kubernetes.
-## Openshift won't deploy with runAsUser: 1000 without additional permissions.
-securityContext:
-  enabled: true  # Should be set to false when running on OpenShift
+## Set to `null` to keep pod level securityContext unspecified
+#podSecurityContext: null
+podSecurityContext:
+  runAsNonRoot: true
+  ## Openshift won't deploy with runAsUser: 1000 without additional permissions.
+  ## Remove the field by setting it to `null`:
+  #runAsUser: null
   runAsUser: 1000
+  seccompProfile:
+    type: "RuntimeDefault"
+
+## Set to `null` to keep container level securityContext unspecified
```

Comment:
please remove that comment.

Reply:
I since removed the comment but I’m unsure whether you or I should mark this discussion as resolved. I’ll go ahead and do it myself given the trivial nature of the requested change but please let me know if it is supposed to be the reviewer marking it as resolved.